test/t8n/t8n.cpp

```diff
@@ -0,0 +1,174 @@
+#include "../state/mpt_hash.hpp"
+#include "../state/rlp.hpp"
+#include "../statetest/statetest.hpp"
+#include <evmone/evmone.h>
+#include <nlohmann/json.hpp>
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <string_view>
+
+namespace fs = std::filesystem;
+namespace json = nlohmann;
+using namespace evmone;
+
+namespace evmone::state
```

Comment:
Do we need this available there or can this be anonymous?

Reply:
Unfortunately it has to be there. Similar to `statetest_runner.cpp`. In other case template functions encode instantiation fails.